client/my-sites/sharing/connections/account-dialog.jsx

```diff
@@ -176,7 +176,8 @@ class AccountDialog extends Component {
 			// If a single account is available, show a simple confirmation
 			// prompt to ask the user to confirm their connection.
 			return this.props.translate(
-				'Confirm this is the account you would like to authorize. Note that your posts will be automatically shared to this account.',
+				"Is this the account you'd like to connect? All your new blog posts will be automatically shared to this account. " +
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?

Reply:
A `context` message like the one on line 182 is normally useful for a short translated string whose meaning might be different in different places: it gives translators who may not see the surrounding code some information about where the string is used. A `comment` explains how the string is used, or what it's trying to say. In the light of the i18n bot's suggestion, we should change the `context` here to a `comment` like `Sharing: asks the user to confirm if they want to share future posts to a connected social media account.`. 

The same `context` is used on the other string with this phrasing, on line 191, so we should change it there too.